hack/newacrtoken/acrtoken.go

```diff
@@ -0,0 +1,65 @@
+package main
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/Azure/azure-sdk-for-go/services/containerregistry/mgmt/2019-06-01-preview/containerregistry"
+	"github.com/Azure/go-autorest/autorest/azure"
+	"github.com/Azure/go-autorest/autorest/date"
+	uuid "github.com/satori/go.uuid"
+	"github.com/sirupsen/logrus"
+
+	"github.com/Azure/ARO-RP/pkg/env"
+)
+
+func main() {
+	ctx := context.Background()
+	log := logrus.NewEntry(logrus.StandardLogger())
+	logrus.SetLevel(logrus.DebugLevel)
+	env, err := env.NewEnv(ctx, log)
+	if err != nil {
+		log.Fatalf("env %v", err)
+	}
+	authorizer, err := env.FPAuthorizer(env.TenantID(), azure.PublicCloud.ResourceManagerEndpoint)
+
+	// https://docs.microsoft.com/en-us/azure/container-registry/container-registry-repository-scoped-permissions
+
+	// az acr token create --name MyToken --registry myregistry  --repository samples/hello-world   content/write content/read
+	tc := containerregistry.NewTokensClient(env.SubscriptionID())
+	tc.Authorizer = authorizer
+	if err != nil {
+		log.Fatalf("authorizer %v", err)
+	}
+	containerregistryResourceGroup := "global"
+	tokenName := os.Getenv("CLUSTER") + "arosvc-pull-all"
```

Comment:
make this a uuid too.

Reply:
we can't start with a number, so I have added some randomness to it.